docs/modules/operation/pages/deep-dive/service-assurance/configuration.adoc

```diff
@@ -58,3 +58,31 @@ During evaluation of an expression, the following scopes are available:
 * Node metadata
 * Interface metadata
 * Service metadata
+
+[[ga-pollerd-configuration-async]]
+== Asynchronous engine
+
+By default, the number of polls active at any given time is limited by the number of threads that are allocated to the daemon.
+Each thread is responsible for one poll, and will block until the poll is completed.
+This is true, even when the polling is done by a Minion at a remote location.
+In this case, the thread will block for the duration of the RPC.
```

Comment:
Here, are we using RPC as an industry-standard term (i.e. Remote Procedure Call)?

```suggestion
In this case, the thread is blocked for the duration of the RPC.
```